test/t/t-2_apply-1_packages-6_aur-2_helpers-7_aura.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source ./lib.bash
+
+# Test AUR functionality using aura.
+
+TestNeedAUR
+TestNeedAURPackage aura 9f00c1bad581ba6a3e96158dfbbabd693a5ee179
```

Comment:
Ah, I see one problem. This Git SHA1 seems to be from the paru AUR package, not aura. Here we can use 32470a0fc37f10148cd480133c5d78e698321e6b for aura v4.0.8.
```suggestion
TestNeedAURPackage aura 32470a0fc37f10148cd480133c5d78e698321e6b
```

Reply:
Ah, I think I lost the fixed sha when I rebased the CI fix attempts. 🤦🏻